tests/Elastic.Markdown.Tests/SiteMap/NavigationTests.cs

```diff
@@ -18,12 +18,11 @@ public void ParsesNestedFoldersAndPrefixesPaths()
 	{
 		Configuration.ImplicitFolders.Should().NotBeNullOrEmpty();
 		Configuration.ImplicitFolders.Should()
-			.Contain("markup")
-			.And.Contain("elastic/observability");
+			.Contain("testing");
```

Comment:
I wanted to be able to test that `testing/req.md` exists as well, but I had trouble figuring out how to do that. After some trial and error, I learned that `Configuration.ImplicitFolders.Should()` only looks at folders (not files). Duh. That makes sense. I then tried doing `Configuration.ImplicitFolders.Files.Should()` but that didn't work. Is there a way to test for the existence of a specific file in an implicit folder? Maybe I'm thinking about this wrong and that's not something we want to test for though.

Reply:
Yeah `ImplicitFolders` is not resolved until `DocumentationSet` is instantiated  at which point we use those folders to glob for `<implicit_folder>/*.md` 

The test was to ensure this list uses full relative paths e.g `a/b/c` not just `c`. 

Will amend the PR to update this.

Reply:
Ahhh okay that makes sense 👍